index.html

```diff
@@ -2326,6 +2327,107 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="comment">
+			<rdef>comment</rdef>
+			<div class="role-description">
+				<p>A comment contains content expressing reaction to other content.</p>
+				<p>Comments can annotate any visible content, from small spans of text, to other comments, to entire articles. Authors SHOULD identify the relationships between comments and the commented content, as follows:</p>
+				<ol>
+					<li>If the comment is a reply to another <code>comment</code>:
+						<ul>
+							<li>If all ancestor comments are available in the DOM, make the reply <code>comment</code> a semantic descendant of the parent <code>comment</code> being replied to, either as a DOM descendant element, or via <pref>aria-owns</pref>.</li>
+							<li>Alternatively, if all ancestor comments are not in the DOM, such as when comments are paginated, the hierarchical
+								level MAY be indicated via <pref>aria-level</pref>. Additional group positional information MAY be indicated via <pref>aria-posinset</pref> and <pref>aria-setsize</pref>.</li>
+						</ul>
+					</li>
+					<li>Otherwise, if the comment relates to other content in the page:
+						<ul>
+							<li>Provide <pref>aria-details</pref> on the commented content, identifying the commented element.</li>
+							<li>If there are many comments relating to the same commented content, either identify all of them individually via <pref>aria-details</pref>, or identify a parent container of the comments, and assign a role of <rref>complementary</rref> to the container.</li>
```

Comment:
Here's why I put it: we're going to provide a computed hint to the screen reader, it would be good to know what role(s) we need to provide that hint on. I think requiring role="complementary" with a computed hint would help have a compromise between having role="commentsection" and not. We essentially still get the benefits of comment section. It's more work for the browser, but have one fewer role.

Reply:
Actually a complementary suggest content that can stand alone, which this isn't.
Perhaps we should limit authors to a named region (if a landmark) or a group (if not a landmark).

Reply:
As discussed on the call, I went with group or region.